packages/firestore/src/core/firestore_client.ts

```diff
@@ -470,6 +470,12 @@ export class FirestoreClient {
             }
           }
         });
+
+        // If the persistence database has been deleted in another client,
+        // shutdown the current client.
```

Comment:
The phrase "persistence database" is imprecise. Also, a comment here could help give larger context, so maybe rephrasing in terms of the user action might make this clearer. Something like:

> When a user calls `clearPersistence()` in one client all other clients need to shut down to allow the delete to succeed.